tests/test_metrics_service.py

```diff
@@ -81,3 +90,18 @@ def test_log(self, mock_execute_ingest, mock_MONGO_CONNECTION, mock_post_event):
         metrics_model = config.get_metric_properties()
         # Log Mixpanel events
         MetricsService.log(config.get_parent_event_name(), metrics_model)
+
+        # Test when user uuid is provided
```

Comment:
More consistent with Rails:

```suggestion
        # Test when metrics uuid is provided
```